packages/oscal-react-library/src/components/OSCALRevision.tsx

```diff
@@ -19,13 +20,17 @@ import {
   StyledTableRow,
 } from "./OSCALSystemImplementationTableStyles";
 
+const NotSpecifiedTypography = styled(Typography)(({ theme }) => ({
+  color: theme.palette.text.disabled,
+})) as typeof Typography;
```

Comment:
Can you put this in a central place and export it? And search for other places where `Not specified` or `No <thing>` appear and use this there?